src/crewai/cli/authentication/constants.py

```diff
@@ -2,7 +2,3 @@
 AUTH0_DOMAIN = "crewai.us.auth0.com"
 AUTH0_CLIENT_ID = "DEVC5Fw6NlRoSzmDCcOhVq85EfLBjKa8"
 AUTH0_AUDIENCE = "https://crewai.us.auth0.com/api/v2/"
```

Comment:
Since you removed WorkOS constants, you can remove auth0 constants as well, especially since we won't be using it anymore.

Reply:
good! please do